cime_config/config_compsets.xml

```diff
@@ -300,16 +316,22 @@
   <compset>
     <alias>I1850Clm40SpCruGs</alias>
     <lname>1850_DATM%CRUv7_CLM40%SP_SICE_SOCN_RTM_SGLC_SWAV</lname>
+    <science_support grid="f09_g17"/>
```

Comment:
I think this should actually refer to the CN version rather than the SP version. The cases actually run were the CN version. We point to the CN startup files for SP, but the actual cases were CN on.